tools/cassandra/README.md

```diff
@@ -7,18 +7,30 @@ How
 - Run make bins
 - You should see an executable `cadence-cassandra-tool`
 
-Setting up initial cassandra schema on a new cluster
+Setting up cassandra schema on a new cluster using Makefile
```

Comment:
Can avoid "using Makefile". It's a bit confusing, Instead, you call it out as a shortcut.